packages/odyssey-storybook/src/components/odyssey-mui/SearchField/SearchField.stories.tsx

```diff
@@ -169,8 +169,17 @@ export const ControlledSearch: StoryObj<typeof SearchField> = {
         setControlledValue(event.target.value),
       [],
     );
+    const onClear = () => {
+      setControlledValue("");
+    };
```

Comment:
This needs to be wrapped in `useCallback`.

Reply:
Why is that? It's only in a story and doesn't depend on anything

Reply:
1. I didn't realize; my bad.
2. I like to put best-practices in our stories because it helps other devs when they copy-paste.

Reply:
All good. I went ahead and wrapped it anyways 😄